lib/go/test/tests/thrifttest_handler.go

```diff
@@ -96,7 +96,7 @@ func (p *ThriftTestHandler) TestStringMap(thing map[string]string) (r map[string
 	return thing, nil
 }
 
-func (p *ThriftTestHandler) TestSet(thing map[int32]struct{}) (r map[int32]struct{}, err error) {
+func (p *ThriftTestHandler) TestSet(thing []int32) (r []int32, err error) {
```

Comment:
This looks like a breaking change; any generated go implementation's handler will need to change.  How will this be communicated?  What happens to existing handlers that implement the older method?